src/WebUI/Common/KubeSummary.tsx

```diff
@@ -285,6 +295,16 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
         return (KubeZeroData._getDefaultZeroData(zeroDataProps));
     }
 
+    private _getDefaultImageLocations(): ImageLocations {
+        return {
+            zeroData: require("../../img/zero-data.svg"),
+            zeroResults: require("../../img/zero-results.svg"),
+            zeroWorkloads: require("../../img/zero-workloads.svg")
+        } as ImageLocations;
+    }
+
+    public static ImageLocations: ImageLocations;
```

Comment:
do not keep this here, have another file or in util, as this does not belong here. we can initialize it in this class, but do not keep. this is same as FF discussion we had